### PR TITLE
React Capstone Project

### PR DESCRIPTION
**Pull request by Dangelo-JAN**
# I created a Cripticooins Metrics SPA using the Original design idea by [Nelson Sakwa on Behance](https://www.behance.net/sakwadesignstudio)

In this milestone I:

- Use React components.
- Use React props.
- Use React Router.
- Connect React and Redux.
- Handle events in a React app.
- Write integration tests with a React testing library.
- Use styles in a React app.
- Use React life cycle methods.
- Use store, actions and reducers in React.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ To get a local copy up and running follow these simple example steps.
 
 ### Prerequisites
 
-Get installed [Node](https://nodejs.org/)
+Get installed [Node](https://nodejs.org/) and [Git](https://git-scm.com/)
 ### Setup
 
 Using your terminal or comand line, run: `git clone https://github.com/Dangelo-JAN/criptocoins-metrics.git` comand
```
Comment (andersonlebon) [inline README.md:29]: - [x] Kindly follow the [project documentation requirements](https://github.com/microverseinc/curriculum-react-redux/blob/main/capstone/react_capstone.md#:~:text=Project%20documentation,description%20of%20the) by recording a video. then provide the link in the `README`

image

- [x] Kindly make sure your project is well deployed and your `README` contains a link to the like version 

image

File: src/components/Coin.jsx [hunk 2]
```diff
@@ -22,7 +22,7 @@ const Coin = (props) => {
         </div>
         <div className="container-column">
             <div className="title">{name}</div>
-            <div className="title">{symbol}</div>
+            <div className="coin-symbol">{symbol}</div>
           </div>
       </div>
```
Comment (andersonlebon) [inline src/components/Coin.jsx]:
```jsx
  const handlerDetails = (id) => {
    window.location.href = `details/${id}`;
  };
```
- [x] Kindly make sure your page is not reloading when I want to show the details of an item as the requirements emphasize showing the details page without reloading the page. you can do it by using the `useNavigate`. check this example [on how to use navigate pages without reloading](https://www.geeksforgeeks.org/reactjs-usenavigate-hook/#:~:text=The%20useNavigate()%20hook%20is,URL%2C%20forward%20or%20backward%20pages.).

Example in your project :point_down: 

```jsx
import React from 'react';
import PropTypes from 'prop-types';
import { useNavigate } from 'react-router-dom';

const Coin = (props) => {
  const navigate = useNavigate();

  const {
    id,
    name,
    image,
    symbol,
  } = props;

  const handlerDetails = (id) => {
    navigate(`/details/${id}`);
  };

  return (
    <>
      <div className="container-column">
        <div className="coin-container">
          <img className="coin-img" src={image} alt={name} />
          <button type="button" className="coin-button" onClick={() => handlerDetails(id)}>-></button>
        </div>
        <div className="container-column">
          <div className="title">{name}</div>
          <div className="coin-symbol">{symbol}</div>
        </div>
      </div>
    </>
  );
};

**Review by andersonlebon — CHANGES_REQUESTED** [review]
## Hi @Dangelo-JAN 

Good job so far! I know you have worked hard on making this milestone But  There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there
## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
## Required Changes ♻️

- [ ] Kindly check comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Coin.jsx (hunk 2 above)
Comment (Dangelo-JAN) [reply src/components/Coin.jsx]: done

File: README.md (hunk 1 above)
Comment (Dangelo-JAN) [reply README.md:29]: done

**Review by arslanbisharat — APPROVED** [review]
Hi,
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- The Pull Request has a proper title and description. ✔️ 
- You have added a descriptive Readme file. Good Job. ✔️ 
- There is no linter error is present. ✔️ 
- Project has met all the technical aspects you needed for this project. Keep rocking! ✔️ 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**